.github/workflows/ci.yml

```diff
@@ -36,7 +36,7 @@ jobs:
       run: dotnet test src --configuration Release --filter Category!=Interactive
 
     - name: Publish
-      run: dotnet publish src/OpenSage.Launcher --configuration Release --runtime ${{ matrix.runtime }} --output build/artifacts/${{ matrix.runtime }}
+      run: dotnet publish src/OpenSage.Launcher --configuration Release --runtime ${{ matrix.runtime }} --output build/artifacts/${{ matrix.runtime }} /p:PublishSingleFile=true /p:PublishTrimmed=true
```

Comment:
looking at a project I set up a while ago that does single-file publish, I also have `-p:DebugType=None` as I think by default it publishes with debugging symbols? [This comment](https://github.com/dotnet/sdk/issues/16975#issuecomment-597612660) also suggests `-p:DebugSymbols=false`